lib/components/user/wizard.tsx

```diff
@@ -93,17 +93,6 @@ class Wizard extends Component<Props> {
         this._routeTo(nextId)
       }
     } else {
-      // Display a toast to acknowledge saved changes
-      // (although in reality, changes quietly took effect in previous screens).
-      toastPromise(
```

Comment:
Also remove the imports of `'../util/toasts'` that are now unused.

Reply:
Good call, thank you!